docs/content/preview/yugabyte-cloud/cloud-secure-clusters/managed-ear.md

```diff
@@ -14,21 +14,18 @@ type: docs
 
 For added security, you can encrypt your clusters (including backups) using a customer managed key (CMK) residing in a cloud provider Key Management Service (KMS). You grant YugabyteDB Managed access to the key with the requisite permissions to perform cryptographic operations using the key to secure the databases in your clusters.
 
-You enable YugabyteDB cluster encryption at rest (EAR) when you create it. See [Create your cluster](../../cloud-basics/create-clusters/).
+You can enable YugabyteDB EAR for a cluster as follows:
```

Comment:
gcp content is the default in prerequisites. Is this intentional ?

Reply:
AWS tab selected by default for me